book/07-git-tools/sections/submodules.asc

```diff
@@ -544,6 +544,8 @@ To https://github.com/chaconinc/MainProject
 As you can see there, Git went into the DbConnector module and pushed it before pushing the main project.
 If that submodule push fails for some reason, the main project push will also fail.
 
+You can enable these behaviors for all future pushes with the `push.recurseSubmodules` config.
+
```

Comment:
It's a good addition, but let's not make it a separate paragraph. How about tacking something like this onto the last paragraph?

> You can make this behavior the default by doing `git config push.recurseSubmodules true`.